bottomless-cli/src/main.rs

```diff
@@ -205,6 +209,14 @@ async fn run() -> Result<()> {
     Ok(())
 }
 
+fn verify_db(path: &PathBuf) -> Result<String> {
```

Comment:
you should verify the result returned by the pragma here, and return a `Result<()>` instead

Reply:
oh right, thats a good idea :) 

I have updated it. Is there any better alternative to `anyhow`, since we plan to remove it

Reply:
I think it can stay inside of bottomless bin, but the alternative to anyhow is to have our own Error enum, and categorize everything manually :)